.github/CODEOWNERS

```diff
@@ -118,6 +118,12 @@
 /sdk/cognitiveservices/cognitiveservices-qnamaker-runtime/ @bingisbestest @nerajput1607
 /sdk/cognitiveservices/cognitiveservices-qnamaker/ @bingisbestest @nerajput1607
 
+# PRLabel: %MixedReality
+/sdk/mixedreality/ @craigtreasure 
+
+# PRLabel: %RemoteRendering
```

Comment:
The label to use is `Remote Rendering`. See https://github.com/Azure/azure-sdk-for-js/labels?q=remote

```suggestion
# PRLabel: %Remote Rendering
```